packages/ui/origin-ui-core/src/components/Modal/ClaimModal.tsx

```diff
@@ -109,7 +108,13 @@ export function ClaimModal(props: IProps) {
         };
 
         const action = isBulkClaim
-            ? requestClaimCertificateBulk({ certificateIds, claimData })
+            ? requestClaimCertificateBulk({
+                  certificateAmounts: certificateIds.map((id) => ({
+                      id,
+                      amount: '0' // TO-DO: Replace 0 with a specific amount
```

Comment:
is that OK?

Reply:
It's OK for now since it will have to change with the UI refactor anyway.

Reply:
Actually, I could change the `amount` to be a specific `number` or some enum `TotalOwnedAmount`.

Reply:
would probably need to write a custom validator for that

Reply:
Addressed this.